build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -79,8 +134,18 @@ def substituteFluentBitPlaceHolders
     windowsFluentBitDisabled = ENV["AZMON_WINDOWS_FLUENT_BIT_DISABLED"]
     kubernetesMetadataCollection = ENV["AZMON_KUBERNETES_METADATA_ENABLED"]
     annotationBasedLogFiltering = ENV["AZMON_ANNOTATION_BASED_LOG_FILTERING"]
+    storageMaxChunksUp = ENV["FBIT_STORAGE_MAX_CHUNKS_UP"]
+    storageType = ENV["FBIT_STORAGE_TYPE"]
```

Comment:
> should we have any validation for storage type? Because that will be something specific right for fluent-bit?

Yes, for all the config parameters and there is an validation . For example, see  - https://github.com/microsoft/Docker-Provider/blob/062c9b4ec9286702e833f6bb39444187f04eacdb/build/common/installer/scripts/tomlparser-agent-config.rb#L252